saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/common/JsonJudgmentImportProcessProcessor.java

```diff
@@ -99,6 +135,35 @@ private void markProcessed(JsonRawSourceJudgment rJudgment) {
         rJudgment.markProcessed();
         rawSourceJudgmentRepository.save(rJudgment);
     }
+    
+    private void handleJudgmentContent(String downloadedContentFilename, Judgment judgment, Judgment oldJudgment) throws IOException {
```

Comment:
I would move this method to a separate service, because it is a separate functionality and it will be easier to test.
JudgmentContentProcessor.processContentFiles(ContentFileTransactionContext, downloadContentFilename, judgment, oldJudgment)